### PR TITLE
Reportback initial markup

### PR DESCRIPTION
**Pull request by weerd**
Initial code for new reportback interface.

Styles will be broken out into smaller modules where applicable and added to Neue. Hard-coded markup will be converted to Drupal theming functions and templates. This is just the initial pass to establish the styles and required markup.

@DFurnes @sbsmith86 

CC: @barryclark @aaronschachter 

For visual reference, desktop view:
reportback layout

PS: all this new markup code is still behind the feature flag.

### CODE REVIEW

**Comment by sbsmith86** [comment]
@weerd Just wondering - what is "flex-gutter()" is this a flexbox thing?

**Comment by weerd** [comment]
Yeah, it's kinda weird and naming is a bit confusing. It's a Bourbon thing as part of their `flex-grid` stuff http://bourbon.io/docs/#flex-grid but unrelated to actual flexbox stuff.

It's the only way I can have Bourbon calculate what the _gutter_ should be without having to use the `span-columns` mixin and having it float the element, etc.

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback.scss [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+  @include media($tablet) {
+    margin-top: 0;
+  }
+}
+
+
+
+// @TODO: Only temp; change class name!
+.gigantor {
```
Comment (sbsmith86) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback.scss:127]: best class name ever. :+1:

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback.scss:127]: LOL... I made the note to change it, but kinda don't want to :grin:

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback.scss [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  background-color: #f00;
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback.scss]: Is this left over from testing something?

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback.scss [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  border: 0;
+  display: block;
+  padding: 0;
+  width: 100%;
+
+  img {
+    display: block;
+    width: 100%;
+    // z-index: 1;
```
Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback.scss]: Should this commented line be removed?

**Comment by DFurnes** [comment]
Looks fantastic. Two little comments. One other suggestion would be maybe adding a little `@TODO: Merge with Neue pattern` line above the two pattern overrides.

File: lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback.scss (hunk 3 above)
Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback.scss]: yep yep!

**Comment by DFurnes** [comment]
Great! :ship:

**Pull request merged**